pkg/mimir/modules.go

```diff
@@ -614,11 +614,18 @@ func (t *Mimir) initQuerier() (serv services.Service, err error) {
 	t.Cfg.Worker.MaxConcurrentRequests = t.Cfg.Querier.EngineConfig.MaxConcurrent
 	t.Cfg.Worker.QuerySchedulerDiscovery = t.Cfg.QueryScheduler.ServiceDiscovery
 
+	var dispatcher *querier.Dispatcher
+
+	if t.Cfg.Querier.QueryEngine == querier.MimirEngine {
```

Comment:
Do we want to have flag for not enabling remote execution query plan, even if we use MQE?

Reply:
Yes, but I'm imagining this will be done in the query-frontend. If the flag is turned off, then the query-frontend will send requests to the querier via the same means as before.